pybossa/sched.py

```diff
@@ -61,7 +62,8 @@ def new_task(project_id, sched, user_id=None, user_ip=None,
         Schedulers.locked: get_locked_task,
         'incremental': get_incremental_task,
         Schedulers.user_pref: get_user_pref_task,
-        'depth_first_all': get_depth_first_all_task
+        'depth_first_all': get_depth_first_all_task,
```

Comment:
Minor comment. If the above constants are defined, the strings could be replaced with the enum values.